Makefile

```diff
@@ -153,6 +156,7 @@ deps: runtime
 		mkdir -p ~/.luarocks; \
 		$(ENV_LUAROCKS) config $(ENV_LUAROCKS_FLAG_LOCAL) variables.OPENSSL_LIBDIR $(addprefix $(ENV_OPENSSL_PREFIX), /lib); \
 		$(ENV_LUAROCKS) config $(ENV_LUAROCKS_FLAG_LOCAL) variables.OPENSSL_INCDIR $(addprefix $(ENV_OPENSSL_PREFIX), /include); \
+		[ '$(ENV_OS_NAME)' == 'darwin' ] && $(ENV_LUAROCKS) config $(ENV_LUAROCKS_FLAG_LOCAL) variables.PCRE_INCDIR $(addprefix $(ENV_PCRE_PREFIX), /include); \
```

Comment:
without `variables. PCRE_DIR` ?

Reply:
![image](https://user-images.githubusercontent.com/40224023/205536385-223556fb-0e57-45a7-a194-038428f0b0d4.png)
Yes,pcre suggests PCRE_DIR or PCRE_INCDIR.I will add PCRE_DIR.

Reply:
try with 

```
[ '$(ENV_OS_NAME)' == 'darwin' ] && $(ENV_LUAROCKS) config $(ENV_LUAROCKS_FLAG_LOCAL) variables.PCRE_DIR $(addprefix $(ENV_PCRE_PREFIX), /lib); \
```

Reply:
form the error message https://github.com/apache/apisix/issues/4945

```
You may have to install PCRE in your system and/or pass PCRE_DIR or PCRE_INCDIR to the luarocks command.
```

sorry, forget what I said, it looks like we only need one of PCRE_DIR or PCRE_INCDIR. 

I test this PR on macOS(amd64), and it works well. 

so we can remove 

```
[ '$(ENV_OS_NAME)' == 'darwin' ] && $(ENV_LUAROCKS) config $(ENV_LUAROCKS_FLAG_LOCAL) variables.PCRE_LIBDIR $(addprefix $(ENV_PCRE_PREFIX), /lib); \
```

BTW, I need to add 

```
   LDAP_DIR = "/usr/local/opt/openldap",
   LDAP_INCDIR = "/usr/local/opt/openldap/include",
```

to `~/.luarocks/config-5.1.lua`

Reply:
All right.I have removed it.